inspectit-gepard-agent/src/test/java/rocks/inspectit/gepard/agent/internal/instrumentation/rules/scopes/InstrumentationScopeTest.java

```diff
@@ -0,0 +1,93 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.agent.internal.instrumentation.rules.scopes;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+import static rocks.inspectit.gepard.agent.testutils.CustomAssertions.assertMethodDescriptionMatcherMatches;
+
+import java.util.Collections;
+import java.util.List;
+import net.bytebuddy.description.type.TypeDescription;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import rocks.inspectit.gepard.agent.internal.instrumentation.model.rules.scopes.InstrumentationScope;
+import rocks.inspectit.gepard.config.model.instrumentation.scopes.ScopeConfiguration;
+
+@ExtendWith(MockitoExtension.class)
+class InstrumentationScopeTest {
```

Comment:
As I mentioned earlier, I´d add tests for interfaces, abstracts etc.

Reply:
Currently, we can't instrument them as mentioned above